packages/strapi-plugin-graphql/services/Mutation.js

```diff
@@ -195,27 +195,49 @@ module.exports = {
 
       // Resolver can be a function. Be also a native resolver or a controller's action.
       if (_.isFunction(resolver)) {
-        context.params = Query.convertToParams(
-          options.input.where || {},
-          (plugin ? strapi.plugins[plugin].models[name] : strapi.models[name])
-            .primaryKey
-        );
-        context.request.body = options.input.data || {};
+        const normalizedName = _.toLower(name);
+
+        let primaryKey;
+
+        if (plugin) {
+          primaryKey = strapi.plugins[plugin].models[normalizedName].primaryKey;
+        } else {
+          primaryKey = strapi.models[normalizedName].primaryKey;
```

Comment:
This will cause an error when the controller doesn't have a model with its name.